packages/fast-components-class-name-contracts-base/src/radio/index.ts

```diff
@@ -2,9 +2,10 @@
  * The class name contract for the radio component
  */
 export interface RadioClassNameContract {
-    radio: string;
-    radio__disabled: string;
-    radio_input: string;
-    radio_label: string;
-    radio_stateIndicator: string;
+    radio?: string;
```

Comment:
Would it be more maintainable if properties on the interface were alphabetically ordered?

Reply:
I tend to order these by type, not alphabetically.